hosting/src/app/games/quiz/lobby/lobby.component.ts

```diff
@@ -48,6 +49,11 @@ export class LobbyComponent implements OnInit, OnDestroy {
                     ...rounds,
                 ];
                 this.availableRounds = this.rounds.map((round) => ({ id: round.uid, title: round.title }));
+                let roundNumber = this.availableRounds.length;
+                if (this.isRoundPreloadNeeded) {
+                    this.questionForm.controls[ 'round' ].setValue(roundNumber > 0 ? this.availableRounds[ roundNumber - 1 ] : undefined);
+                    this.isRoundPreloadNeeded = false;
+                }
```

Comment:
Would be cleaner to read like this... could even have the `get('round')` abstracted out

```
if (this.availableRounds.length && !this.questionForm.get('round').value) {
    this.questionForm.get('round').setValue(this.availableRounds[ this.availableRounds.length - 1 ]);
}
```